### PR TITLE
planner_cspace_msgs: define a Move action having tolerance parameters

### PR DESCRIPTION
**Pull request by DaikiMaekawa**

### CODE REVIEW

**Comment by at-wat** [comment]
## [#58] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [#59] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

File: planner_cspace_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -2,11 +2,20 @@ cmake_minimum_required(VERSION 2.8.3)
...
   std_msgs
 )
 
 find_package(catkin REQUIRED COMPONENTS message_generation ${MESSAGE_DEPENDS})
 
+add_action_files(
+  DIRECTORY
+    action
+  FILES
+    MoveFlex.action
```
Comment (at-wat) [inline planner_cspace_msgs/CMakeLists.txt]: I think `MoveWithTolerance.action` would be more descriptive.

File: planner_cspace_msgs/action/MoveFlex.action [hunk 2]
```diff
@@ -0,0 +1,7 @@
+geometry_msgs/PoseStamped target_pose
+float64 goal_tolerance_lin
+float64 goal_tolerance_ang
+float64 goal_tolerance_ang_finish
```
Comment (at-wat) [inline planner_cspace_msgs/action/MoveFlex.action]: Could you add a comment to explain difference between `goal_tolerance_ang` and `goal_tolerance_ang_finish`?

**Comment by at-wat** [comment]
## [#60] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by DaikiMaekawa** [comment]
Addressed your comments.

**Pull request merged**